### DISCUSSION TITLE
Conversion to CustomTkinter

### DISCUSSION THREAD

**Discussion by DylanKirbs** [PaulleDemon/tkVideoPlayer#23] [Ideas]
Hi, would you consider the use of [CustomTkinter](https://github.com/TomSchimansky/CustomTkinter) as it makes the UI a lot more customisable and generally neater.
I understand it would incur additional dependencies, but it is worth it in my opinion.
I'd be happy to help out with part of the conversion or making a cloned version for CTk.

**Comment by PaulleDemon** [comment]
Hi @DylanKirbs, you can easily integrate tkvideoplayer to CustomTkinter; see the example below.

```
import customtkinter
from tkVideoPlayer import TkinterVideo

app = customtkinter.CTk()  # create CTk window like you do with the Tk window
app.geometry("400x240")

def button_function():
    video_player.play()

video_player = TkinterVideo(master=app, scaled=True)
video_player.pack(expand=True, fill='both')
video_player.load("test.mp4")

button = customtkinter.CTkButton(master=app, text="Play", command=button_function)
button.pack()

app.mainloop()
```

If you found this helpful, mark this as correct, if you have any further question feel free to ask.

**Reply by DylanKirbs** [reply]
Ah, I misunderstood something from the demo on PyPi, I thought the GUI for the control bar was built into the module. I see now that I was mistaken, but that could also be a suggestion. 😅

**Reply by CodeYan01** [reply]
Not really trying to use ctk, but i was also confused by the image on PyPi, i thought the controls were already part of the widget, and saw no options in the init to not include controls. Perhaps the PyPi description should be made clearer too
